queries/slang/highlights.scm

```diff
@@ -0,0 +1,42 @@
+; inherits: hlsl
+
+[
+ "var"
+ "let"
+ "This"
```

Comment:
is `This` like js/ts? maybe `@variable.builtin` would be nicer then (I could be completely wrong)